API.md

```diff
@@ -115,6 +115,8 @@ Returns a `Promise` that resolves to [a database instance](#store-api). `name` (
 
 - `overwrite` (boolean): Overwrite an existing database (Default: `false`)
 - `replicate` (boolean): Replicate the database with peers, requires IPFS PubSub. (Default: `true`)
+- `meta` (object): An object to be added to the database manifest (learn more about the [db manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Data in the manifest affects the address of the db and cannot be changed after creation. It's recommended to store smaller objects and must be serializable to json. Read the object at instance property `db.options.meta`. Default: `undefined`.
```

Comment:
How about:

- `meta` (object): An optional object [database manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Immutably stores any JSON-serializable key/value pair and outputs via `db.options.meta`. Default: `undefined`.

Reply:
There's also the idea that these things will affect the DB address but I'm not sure if here specifically is the place to put that info

Reply:
much more concise which is nice, i would slightly rather `key/value pair` -> `value` and maybe `outputs` -> `readable`?

Reply:
yeah that's fine